graphql/src/graphql.ml

```diff
@@ -1,6 +1,15 @@
-module Schema = Graphql_schema.Make(struct
+module Io = struct
   type +'a t = 'a
 
   let bind x f = f x
   let return x = x
-end)
\ No newline at end of file
+end
+
+module Stream = struct
+  type +'a io = 'a Io.t
+  type 'a t = 'a list
+
+  let map x f = List.map f x
+end
+
+module Schema = Graphql_schema.Make (Io) (Stream)
```

Comment:
I think there should be consistency in how the functor `Graphql_schema.Make` is applied for `graphql.ml` (named modules), `graphql_lwt.ml` and `graphql_async.ml` (unnamed modules).